internal/cli/operation.go

```diff
@@ -113,7 +114,7 @@ func (m merge) Execute(w io.Writer) error {
 		return err
 	}
 
-	yaml, err := out.AsValue().ToYAML()
+	yaml, err := yaml.Marshal(out.AsValue().Interface())
```

Comment:
Hm, did this line have to change? (sorry for all these comments, but keeping the diff down is also nice to any users...)